mache-couchbase/src/main/java/com/excelian/mache/couchbase/builder/CouchbaseProvisioner.java

```diff
@@ -97,17 +67,27 @@ public static ClusterBuilder couchbase() {
      * A builder with defaults for a Couchbase cluster.
      */
     public static class CouchbaseProvisionerBuilder {
-        private final ConnectionContext<Cluster> connectionContext;
         private final BucketSettings bucketSettings;
+        private CouchbaseEnvironment couchbaseEnvironment = DefaultCouchbaseEnvironment.create();
```

Comment:
Actually this is bad - we may need to force users to provide a couchbase env - if we always create it here and then the user creates their own, they're always going to get the warning in the logs. maybe add this one to the mandated props, or in the create below, have if null then env.create?